src/index.tsx

```diff
@@ -40,14 +37,16 @@ export const RainbowButton = ({
   );
 };
 
+export const assets = {
+  rainbow_icon: require('./public/images/rainbow-icon.png'),
```

Comment:
Requiring a path to a png, does this assume that the consumer has an image loader to support this?